distribution/docs/src/main/resources/_contents/quickstartcontents.adoc

```diff
@@ -0,0 +1,122 @@
+
+This quick tutorial will enable install, configuring and using a basic instance of ${branding}.
+
+These steps will demonstrate:
+
+- [*] Prerequisites
+- [*] Installation
+- [*] Ingesting into the ${ddf-branding} Catalog
+
+==== Prerequisites
+
+* Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+* Supported platforms are *NIX - Unix/Linux/OSX, Solaris, and Windows.
+* http://www.oracle.com/technetwork/java/javase/downloads/index.html[JDK8] must be installed.
+* The `JAVA_HOME` environment variable must be set to the location where the JDK is installed.
+
+.Setting `JAVA_HOME` on *NIX
+----
+JAVA_HOME=/usr/java/jdk1.8.0
+export JAVA_HOME
+----
+
+.Setting `JAVA_HOME` on Windows
+----
+set JAVA_HOME=C:\Program Files\Java\jdk1.8.0
+----
+
+.*NIX
+[WARNING]
+====
+Unlink `/usr/bin/java` if it is already linked to a previous version of the JRE:
+`unlink /usr/bin/java`
+====
+
+[TIP]
+====
+Verify that the `JAVA_HOME` was set correctly.
+
+.*NIX
+----
+echo $JAVA_HOME
```

Comment:
Well it depends who's reading this... if the use case here is that the user understands that `echo` will only succeed in telling him that the variable has been set (and show him what it has been set to), but not in telling him that the value is correct... then `echo` is fine :)
Assuming these operations are being done by an Admin type fellow then I think it's safe...